Main.java

```diff
@@ -11,7 +11,10 @@ public static void main(String[] args) {
 
     // Assigned to Athena
     public static int getSum(int[] arr) {
-        return 0;
+        int sum = 0;
+        for (int i = 0; i < arr.length; i++)
+            sum=sum+arr[i];
+            return sum;
```

Comment:
For consistency, I'd like to always use curly brackets for code blocks when in doubt. In this case have a curly bracket around the contents of the for loop:
```suggestion
        for (int i = 0; i < arr.length; i++) {
            sum=sum+arr[i];
        }
        return sum;
```